Jenkinsfile

```diff
@@ -1,10 +1,50 @@
 pipeline {
-  agent any
-
+  agent {
+    kubernetes {
+      label 'xtext-build-pod'
+      defaultContainer 'jnlp'
+      yaml '''
+apiVersion: v1
+kind: Pod
+spec:
+  containers:
+  - name: jnlp
+    image: 'eclipsecbi/jenkins-jnlp-agent'
+    args: ['\$(JENKINS_SECRET)', '\$(JENKINS_NAME)']
+    volumeMounts:
+    - mountPath: /home/jenkins/.ssh
+      name: volume-known-hosts
+  - name: xtext-buildenv
+    image: docker.io/smoht/xtext-buildenv:0.7
```

Comment:
i assume we should get rid of this one correct?
do we need it for jekyll?

Reply:
yes. we should build an image just for this purpose owned by Xtext.